test/extensions/config_subscription/grpc/grpc_mux_failover_test.cc

```diff
@@ -16,12 +16,14 @@ namespace Envoy {
 namespace Config {
 namespace {
 
-class GrpcMuxFailoverTest : public testing::Test {
```

Comment:
Coverage build is failing due to coverage regression, it would be good to verify all the new, somewhat intricate, code paths have full coverage.